### ISSUE TITLE
Line number is errors is 0 based, should be 1 based.

### ISSUE THREAD

**Issue by billw2012** [SQFvm/vm#36]
1 based is the standard in code line error reporting I believe.
It is the last issue stopping my Visual Studio Code compile task from correctly populating the problems tab!

**Comment by billw2012** [comment]
It looks like the line numbers also exclude comment lines. As in if there is a multi line comment 3 lines long before the error line, the reported line number is 4 less that it should be: -1 because it is 0 based, and -3 for the ignored comment lines. Error that should show on line 10 shows on line 6.

**Comment by X39** [comment]
Cannot reproduce the multi-line comment part

can you provide example code where this happens?

**Comment by billw2012** [comment]
I was wrong with the description it appears on multiline comments over more then a single line cause the problem.
```
// comment
/* comment
*
*
*
*/
test = { 
	error ?
};
```
gives error message:
```
 Executing task: sqfvm.exe -a --parse-only --load ./../../Project_0.Altis -i "c:\Users\billw\Documents\Arma 3\mpmissions\Project_0\Project_0.Altis\UI\test.sqf" <

        error ?
       ^
[ERR][L4|C7|c:\Users\billw\Documents\Arma 3\mpmissions\Project_0\Project_0.Altis\UI\test.sqf]   Expected '}'.
The terminal process terminated with exit code: 4294967295
```
Note it says L4, it should be L8.

**Comment by X39** [comment]
Fixed in 7c49585891ca6c7f7c0a0f1e32b74c85bfeaa2c0
